.github/workflows/e2e-test.yml

```diff
@@ -798,7 +793,7 @@ jobs:
 
       - name: Install KServe
         run: |
-          ./test/scripts/gh-actions/setup-kserve.sh "raw" ${{ matrix.network-layer }}
+          ./test/scripts/gh-actions/setup-kserve.sh "Standard" ${{ matrix.network-layer }}
```

Comment:
all lowercase, maybe?

Reply:
This change was intentional as well.
Instead of converting everything to lowercase and then changing it back internally, I thought it would be better to just use the values as they are.
After all, setup-kserve.sh is mainly for CI purposes and is only used by community members, so I don’t think it’ll cause any major issues.